### PR TITLE
PAYARA-2864 Document Payara NetBeans plugins v1.3

### PR DESCRIPTION
**Pull request by jGauravGupta**

### CODE REVIEW

**Comment by jGauravGupta** [comment]
Do not merge until plugin verification is not completed : http://plugins.netbeans.org/plugin/72263/payara-server

File: documentation/ecosystem/netbeans-plugin/README.adoc [hunk 1]
```diff
@@ -0,0 +1,54 @@
+= Payara NetBeans Plugin
```
Comment (OndrejM) [inline documentation/ecosystem/netbeans-plugin/README.adoc]: All new files and renamed files must be added to SUMAMRY.adoc

File: documentation/ecosystem/netbeans-plugin/README.adoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Payara Micro and Server now has a NetBeans plugin to allow you to add Payara Server as a
```
Comment (OndrejM) [inline documentation/ecosystem/netbeans-plugin/README.adoc]: This should be "Payara Platform provides the Netbeans plugins to integrate Payara Server and Payara Micro into Netbeans. The plugins are installable from the NetBeans interface". 

And further information should be added below for each plugin, see my other comments

File: documentation/ecosystem/netbeans-plugin/README.adoc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+recognised server in NetBeans. This means you can deploy and manage applications
+from the NetBeans interface. The plugin is installable from the NetBeans
+interface.
+
+In version {{ book.currentVersion }}, the available plugins are :
+
+* link:payara-server.adoc[Payara Server]
```
Comment (OndrejM) [inline documentation/ecosystem/netbeans-plugin/README.adoc]: Move the details from above here:

* Payara Server - allows you to add Payara Server as a recognised server in NetBeans. This means you can deploy and manage applications from the NetBeans interface.
* Payara Micro - allows you to use Payara Micro to start and reload your applications from the Netbeans interface

File: documentation/ecosystem/netbeans-plugin/README.adoc [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+
+[[installing]]
+== Installing Payara NetBeans Plugin
+
+Payara offers three ways to install NetBeans plugins :
+
+=== Manual Installation
+
+1. To download the Payara NetBeans Plugin, Please http://plugins.netbeans.org/plugin/72263/payara-server[click here].
```
Comment (OndrejM) [inline documentation/ecosystem/netbeans-plugin/README.adoc]: , **p**lease (small p)

File: documentation/ecosystem/netbeans-plugin/README.adoc [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+interface.
+
+In version {{ book.currentVersion }}, the available plugins are :
+
+* link:payara-server.adoc[Payara Server]
+* link:payara-micro.adoc[Payara Micro]
+
+
+[[installing]]
+== Installing Payara NetBeans Plugin
```
Comment (OndrejM) [inline documentation/ecosystem/netbeans-plugin/README.adoc]: Plugin**s**

File: documentation/ecosystem/netbeans-plugin/README.adoc [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+2. To get the plugins ready for installation, start NetBeans IDE as described in https://netbeans.org/community/releases/81/install.html[Installing and the Starting the NetBeans IDE].
+3. Extract the nbm file(s) from PayaraPlugins.zip file.
+4. Start NetBeans and select the *Tools -> Plugins* menu item.
+5. Switch to the "Downloaded" tab and select the *Add Plugins...* button to browse to the downloaded nbm files.
+6. Once the file has been selected, NetBeans will display module information. Now click the *Install* button at the bottom.
+7. When the NetBeans IDE Installer screen is displayed, click next and accept the license agreements and click Install.
+8. The plugin has now been installed and can be seen in the "Installed" tab. Now click *Finish* to restart NetBeans IDE.
+
+
+=== Update Center Installation
```
Comment (OndrejM) [inline documentation/ecosystem/netbeans-plugin/README.adoc]: I would move "Update Center" section before manual installation

**Comment by jGauravGupta** [comment]
Plugin is now approved .

**Pull request merged**